### PR TITLE
update troubleshooting + quickstart docs to aid with capi incompatibilities

### PR DESCRIPTION
**Pull request by pborn-ionos**

### CODE REVIEW

File: docs/Usage.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ the `EXP_CLUSTER_RESOURCE_SET` is required if you want to deploy CNI using clust
 
 Once you have access to a management cluster, you can initialize Cluster API with the following:
 ```
-clusterctl init --infrastructure proxmox --ipam in-cluster
+clusterctl init --infrastructure proxmox --ipam in-cluster --core cluster-api:v1.5.3
```
Comment (mcbenjemaa) [inline docs/Usage.md:104]: @avorima 
Do you agree with pinning the capi version in the quick-start docs?
while our compatibility versions matrix mentions that v1.6 not yet supported?
https://github.com/ionos-cloud/cluster-api-provider-proxmox/tree/main?tab=readme-ov-file#compatibility-with-cluster-api-and-kubernetes-versions


In my point of view we should plan the support for v1.6 very soon.

Comment (mcbenjemaa) [reply docs/Usage.md:104]: Let's wait for @avorima feedback.

Comment (avorima) [reply docs/Usage.md:104]: Yes, since those commands are meant to be copied into a terminal.

Comment (wikkyk) [reply docs/Usage.md:104]: ok then

**Review by wikkyk — APPROVED** [review]

**Pull request merged**